src/test/java/mate/academy/dao/RoleDaoImplTest.java

```diff
@@ -0,0 +1,55 @@
+package mate.academy.dao;
+
+import mate.academy.dao.impl.RoleDaoImpl;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RoleDaoImplTest extends AbstractTest {
+    private RoleDao roleDao;
+    private Role role;
+
+    @Override
+    protected Class<?>[] entities() {
+        return new Class[]{Role.class};
+    }
+
+    @BeforeEach
+    void setup() {
+        role = new Role();
+        role.setRoleName(Role.RoleName.USER);
+        roleDao = new RoleDaoImpl(getSessionFactory());
+    }
+
+    @Test
+    void save_ok() {
+        Role actual = roleDao.save(role);
+        Assertions.assertNotNull(actual);
+        Assertions.assertEquals(1L, actual.getId());
+    }
+
+    @Test
+    void save_notOk() {
```

Comment:
Please add to naming which problem should produce negative result.
Please recheck all occurences